### DISCUSSION TITLE
Rate Limiting

### DISCUSSION THREAD

**Discussion by NEMA-IT** [Jericho/StrongGrid#445] [General]
Does the StrongGrid client automatically delay & retry when SendGrid responds with rate-limiting error? (status code == 429). If it's my responsibility, what's the recommended approach? (For example, how do I access the status code in the response?)

**Comment by Jericho** [comment]
StrongGrid has a built-in strategy to handle rate-limiting exception. This strategy takes care of waiting a reasonable amount of time before re-issuing your request.

**Reply by NEMA-IT** [reply]
Awesome! Thank you so much. I've just started using StrongGrid and it's really impressive.
